### DISCUSSION TITLE
[PvP] Increse weekly AP reward from 2s and 3s brackets

### DISCUSSION THREAD

**Discussion by Ottawia** [chromiecraft/chromiecraft#7618] [Suggestions]
Solo Queue has been added to ChromieCraft, and it has negatively impacted the other arena brackets because it provides more Arena Points (AP) than 2s and 3s. As a result, there are now barely any teams, and even fewer players are able to complete their weekly games. Our usual day for arena activity, when most players participated (Tuesday), is now dominated solely by Solo Queue.

This is harmful to overall PvP activity. Don’t get me wrong—Solo Queue can stay—but at the moment, it is the only viable arena activity, and participation is quite low. Issues like class stacking, healer stacking, or matches without any healers further exacerbate the situation (more details here: https://discord.com/channels/788063059926712341/801844176416276500/1316534129138995260).

Boosting the AP rewards for 2s and 3s could help revitalize these brackets. This would give players who seek more of a challenge—rather than relying on luck (and yes, Solo Queue is largely luck-based at the moment)—a fair chance to compete without falling behind those who focus solely on Solo Queue. Importantly, this change wouldn’t reduce the AP provided by Solo Queue, so it wouldn’t harm more casual players who prefer playing only Solo Queue.

The difference between 3s and 3 Solo is 0.08%, which is negligible, especially when there are no games happening. If that number were increased, perhaps players would put more effort into playing 3s with non-random teams (avoiding setups like triple healers). I also believe there would be players who utilize CCs and other PvP strategies more effectively in such matches.
The difference between 2s and Solo Queue is simply ridiculous. An arena where you face a coordinated team (rather than a random one) should grant more AP, as it is usually more challenging to defeat a non-random team.

**Comment by minthes** [comment]
I great way to increase participation and give players more arena points from 2s/3s would be to implemenet a daily quest that rewards a small amount of arena points per day that is not so big to be impactful if players miss a day or 2 but over time it adds up (for example: 25AP per day).

**Comment by tbcxd** [comment]
Fully support this
